### PR TITLE
fix: broaden operations course revenue metrics

### PR DESCRIPTION
**Pull request by QinyingShi**
## What
- broaden operations course detail revenue aggregation to use `paid_price` first and fall back to `payable_price` for successful orders
- remove the coupon/promo-only gating so externally redeemed revenue is included in the course income view
- add focused backend tests for cross-channel successful orders and zero-value activation orders

## Why
The goal of this metric is to understand course revenue more accurately, not only revenue settled directly inside the platform.

Coupon or promo redemption can still represent real course revenue when users purchased redemption rights through an external channel and then redeemed on the platform. Restricting the fallback to coupon/promo linkage undercounts this type of income, and it also misses other successful external-channel orders where `paid_price` is zero but `payable_price` still records the revenue amount.

With this change, the metric behaves as:
- use `paid_price` when it is present
- otherwise use `payable_price` for successful orders
- keep zero-value activation / authorization orders counted in order volume but excluded from revenue

## Tests
- `cd src/api && LOGGING_PATH=logs/local-debug.log ./.venv/bin/pytest -q tests/service/shifu/test_admin_course_detail.py`
- `pre-commit run -a`

## Summary by CodeRabbit

* **Changes**
  * Order amount calculation simplified. System now uses paid price when available; otherwise, falls back to payable price for metrics reporting.
  * Updated order metrics documentation to reflect current calculation methodology.

### CODE REVIEW

File: src/api/flaskr/service/shifu/admin_dtos.py [hunk 1]
```diff
@@ -204,7 +204,7 @@ class AdminOperationCourseDetailMetricsDTO(BaseModel):
     order_count: int = Field(..., description="Successful order count", required=False)
     order_amount: str = Field(
         ...,
-        description="Collected amount including full coupon and promo redemptions",
+        description="Collected amount for successful orders using paid price when present, otherwise payable price",
```
[the remark this answers is not in the document]
Comment (QinyingShi) [reply src/api/flaskr/service/shifu/admin_dtos.py]: Good catch. I updated the DTO description to match the exact fallback rule in the implementation: use `paid_price` when `paid_price > 0`, otherwise use `payable_price` when `payable_price > 0`.

**Comment by QinyingShi** [comment]
Backend check note:

- The failing check is the full backend suite (`python -m pytest --testmon tests`).
- This PR's relevant file `tests/service/shifu/test_admin_course_detail.py` passed in CI.
- `main` is currently showing the same backend-suite failure pattern as well, with unrelated failures concentrated in `dashboard`, `learn`, `order`, and `shifu_utils`.
- Treating the red backend check here as a pre-existing baseline issue rather than a regression introduced by this PR.

For this PR specifically, the revenue-metrics change and its targeted coverage are passing.
